Dockerfile.centos

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2012-2016 Red Hat, Inc
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#   Dharmit Shah  - Initial implementation
+#   Mario Loriedo - Improvements
+#
+# To build it, run in the repository root:
+#  `docker build -t registry.centos.org/eclipse/che-server -f Dockerfile.centos .`
```

Comment:
Right, if it's being built locally, a user probably doesn't need it. But for che-launcher image to be able to find it properly on local setup, it would be a good idea to name/tag it that way. Let me know if that understanding is incorrect. 

Also, I've taken a cue from the build statement [here](https://github.com/eclipse/che/blob/master/Dockerfile#L12).